packages/cli/src/commands/ui/create/angular.spec.ts

```diff
@@ -19,6 +20,7 @@ import {Config, Configuration} from '../../../lib/config/config';
 import {
   IsNpmVersionInRange,
   IsNodeVersionInRange,
+  hasNecessaryCoveoPrivileges,
```

Comment:
Either uniformize the other preconditions in term of naming (e.g. `IsNpmVersionInRange`->`isNpmVersionInRange`) or follow what's in place ;)